assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -105,6 +106,21 @@ export const TIPS = {
       ),
     ],
   },
+  [KEYS.EMBED_WEB_STORIES]: {
+    title: __('Embed Web Stories', 'web-stories'),
+    /* `figureSrcImg` and `figureAlt` are temporary until
+    we get an animation. Once we have the animation then these
+    attributes should be removed. */
+    figureSrcImg: 'images/help-center/story_embed_module_1',
+    figureAlt: __('Embed web story graphic', 'web-stories'),
+    description: [
+      __(
+        'When you create a new post in WordPress, you can embed any of your Stories. <a>Learn more</a>',
+        'web-stories'
+      ),
+    ],
+    href: 'https://wordpress.org/support/article/embeds/',
```

Comment:
Let's make this translatable:

```suggestion
    href: __('https://wordpress.org/support/article/embeds/', 'web-stories')
```

Reply:
are links supposed to be translatable or should we add a query parameter to determine the language?

Reply:
Wordpress links are translatable, if you look for `wordpress.` you'll see a handful getting translated.

Reply:
Ah good call, found them!